bin/hdfcoinc/pycbc_coinc_hdfinjfind

```diff
@@ -84,11 +87,19 @@ for trigger_file, injection_file in zip(args.trigger_files,
     if 'foreground/time1' in f:
         multi_ifo_style = False
         ifo_list = ['H1', 'L1']
+        assert args.min_required_ifos > 2
```

Comment:
I don't think this is what `assert` does. Read the documentation for it

Reply:
ah yes - that was because I had been asserting, then I realised raising an error would be more descriptive. forgot to change to an if statement